app/src/features/authentication/signup/services/apiConfirmCode.ts

```diff
@@ -6,7 +6,7 @@ export async function sendEmailVerification(email: string) {
     headers: {
       "Content-Type": "application/json",
     },
-    body: JSON.stringify(email),
+    body: JSON.stringify({ email: email }),
```

Comment:
you can use JSON.stringify({email}) as they have the same name

Reply:
it wasn't working for some reason